src/mvs_eland/C0_data_processing.py

```diff
@@ -1710,8 +1711,13 @@ def add_maximum_cap(dict_values, group, asset, subasset=None):
     else:
         dict = dict_values[group][asset][subasset]
     if MAXIMUM_CAP in dict:
-        # check if maximumCap is greater that installedCap
         if dict[MAXIMUM_CAP][VALUE] is not None:
+            # adapt maximumCap of nocolan-dispatchable sources
+            if group == ENERGY_PRODUCTION and dict[RENEWABLE_ASSET_BOOL][VALUE] == True:
+                dict[MAXIMUM_CAP][VALUE] = dict[MAXIMUM_CAP][VALUE] / max(
```

Comment:
Maybe add a logging.debug() message here, that the maxcap needed to be adapted as the timeseries was !1. And that this is done as the constraint is to limit the power, not the flow.

Reply:
done